solr/solr-ref-guide/build.gradle

```diff
@@ -38,6 +38,17 @@ versionsLock {
 
 description = 'Solr reference guide'
 
+def shouldSkipTasks = propertyOrEnvOrDefault("solr.ref-guide.skip", "SOLR_REF_GUIDE_SKIP", "false")
+if (shouldSkipTasks.toLowerCase().startsWith("t")) {
```

Comment:
it should really parse this as a boolean. Also, I think a proper way to skip ref-guide tasks would be to eliminate the dependency on the artifacts of that project (if the trigger is enabled), not just disable all of the project's tasks.

Reply:
Hmm... I looked into it and the ref guide is not an artifact that is sucked in during the regular build:
{code}
gradlew -p solr/packaging assemble
{code}

I understand it does run when you're running {{gradlew assemble}} from the top level but this is actually what it's supposed to do - build all subprojects. The proper way to limit the build to what you need is to point at the tasks (or projects) you want to be built - then anything else is automatically ignored?